From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "benh@amazon.com" <benh@amazon.com>
Date: Tue, 18 May 2021 13:17:31 +1000
Subject: [PATCH] manual: Remove warning in the documentation of the abort
 function

Please see the following bug for a complete summary: https://bugzilla.redhat.com/show_bug.cgi?id=1615608

(cherry-picked from 7f0e1933f08d52b22b02fdcd075382506f7b2e9c)

diff --git a/manual/startup.texi b/manual/startup.texi
index 7395d32dd0c877487938857fd432650e87fe7bb2..21c48cd0374dcc906c33165267412081fc7de2dc 100644
--- a/manual/startup.texi
+++ b/manual/startup.texi
@@ -1005,14 +1005,6 @@ This function actually terminates the process by raising a
 intercept this signal; see @ref{Signal Handling}.
 @end deftypefun
 
-@c Put in by rms.  Don't remove.
-@cartouche
-@strong{Future Change Warning:} Proposed Federal censorship regulations
-may prohibit us from giving you information about the possibility of
-calling this function.  We would be required to say that this is not an
-acceptable way of terminating a program.
-@end cartouche
-
 @node Termination Internals
 @subsection Termination Internals
 
